### PR TITLE
Use opw_kinematics instead of trac-ik for IRB1200

### PR DESCRIPTION
**Pull request by cjue**

### CODE REVIEW

File: abb_irb1200_5_90_moveit_config/config/kinematics.yaml [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+  kinematics_solver: moveit_opw_kinematics_plugin/MoveItOPWKinematicsPlugin
+  opw_kinematics_geometric_parameters:
+      a1:  0
+      a2: -0.042
+      b:   0.0
+      c1:  0.3991
+      c2:  0.448
+      c3:  0.451
+      c4:  0.082
+  opw_kinematics_joint_offsets: [0.0, 0.0, -1.57079632679, 0.0, 0.0, 0.0]
```
Comment (gavanderhoorn) [inline abb_irb1200_5_90_moveit_config/config/kinematics.yaml]: Any reason not to use `deg(..)` here?

**Review by gavanderhoorn — DISMISSED** [review]
Looks OK to me.

I've left one in-line comment.

If you can confirm you've verified these parameters are correct, we can merge.

File: abb_irb1200_5_90_moveit_config/config/kinematics.yaml (hunk 1 above)
Comment (cjue) [reply abb_irb1200_5_90_moveit_config/config/kinematics.yaml]: No, thanks for the reminder, added a commit with `deg()` syntax.

**Comment by cjue** [comment]
> If you can confirm you've verified these parameters are correct, we can merge.

I can confirm the parameters, these configs have already been used with real robots.

**Comment by gavanderhoorn** [comment]
Closing and re-opening to nudge Actions.

**Comment by gavanderhoorn** [comment]
And a comment: should the package manifests of both packages not be updated to state the dependency on the OPW Kinematics MoveIt plugin now, instead of trac ik?

**Comment by gavanderhoorn** [comment]
thanks for keeping the `j23_coupled` commits out of this. It would be unrelated.

**Comment by cjue** [comment]
The irb1200 config are IMHO already correct regarding `j23_coupled`. They have the opposite setting as compared to the irb1600 support packages.

**Comment by cjue** [comment]
I rebased this to the current `kinetic-devel`

**Review by gavanderhoorn — APPROVED** [review]
Thanks for the PR :+1:

**Pull request merged**